docs/platforms/python/tracing/instrumentation/custom-instrumentation/index.mdx

```diff
@@ -170,7 +198,7 @@ def eat_pizza(pizza):
 
 ## Accessing the Current Span
 
-To change data in the current span, use `sentry_sdk.Hub.current.scope.span`. This property will return a span if there's one running, otherwise it will return `None`.
+To change data in the current span, use ` sentry_sdk.get_current_scope().span`. This property will return a span if there's one running, otherwise it will return `None`.
```

Comment:
```suggestion
To change data in the current span, use ` sentry_sdk.get_current_span()`. This function will return a span if there's one running, otherwise it will return `None`.
```